test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerFhirStorageTestHelper.cs

```diff
@@ -211,7 +211,7 @@ public async Task ExecuteSqlCmd(string sql)
 
         public async Task DeleteDatabase(string databaseName, CancellationToken cancellationToken = default)
         {
-            if (!DropDatabase)
+            if (IsAzure())
```

Comment:
Why don't we attempt to delete database when in Azure?

Reply:
Because tests will start timing out, even though I decreased command timeout to 15 seconds.